Mathlib/Tactic/NormNum/BigOperators.lean

```diff
@@ -0,0 +1,372 @@
+/-
+Copyright (c) 2023 Anne Baanen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen, Floris van Doorn
+-/
```

Comment:
Could you add a header here so the porting dashboard marks off `algebra.big_operators.norm_num` as done on account of this PR?